lib/src/components/NxSubmitMask/NxSubmitMask.scss

```diff
@@ -26,21 +27,29 @@
 
 .nx-submit-mask--fullscreen {
   position: fixed;
+
+  .nx-submit-mask__message {
+    position: absolute;
```

Comment:
It seems slightly weird that the positioning of this element is done absolutely in this case, but using flexbox in the normal case.  Could use `align-self` and `margin-top` to keep it as flexbox in both cases.

Reply:
> It seems slightly weird that the positioning of this element is done absolutely in this case, but using flexbox in the normal case. Could use `align-self` and `margin-top` to keep it as flexbox in both cases.

Flexbox aligning: https://github.com/sonatype/sonatype-react-shared-components/pull/182/commits/457c922130aba6edad12f7121732a4530f493164